modules/storages/spec/features/admin_storages_spec.rb

```diff
@@ -537,7 +537,8 @@
     end
   end
 
-  it 'creates, edits and deletes storages', :webmock do
+  # skipped to be revised later. broken due to removal of storage_primer_design feature flag
+  xit 'creates, edits and deletes storages', :webmock do
```

Comment:
@ba1ash we can delete these specs, they're for the old flow

Reply:
Sure, I will do it. I postponed it because I was not sure. Thanks for clarifying!